scripts/pull_github_pr.sh

```diff
@@ -28,11 +61,6 @@ if [[ -z "$JENKINS_USERNAME" || -z "$JENKINS_API_TOKEN" ]]; then
   exit 1
 fi
 
-if [ -z "$1" ]; then
-	echo Please provide a github pull request number
-	exit 1
-fi
```

Comment:
We still need this check, you should check `$PR_NUM` now.

Reply:
Added